noxfiles/dev_nox.py

```diff
@@ -170,6 +171,8 @@ def fides_env(session: Session, fides_image: Literal["test", "dev"] = "test") ->
     session.run(
         *START_TEST_ENV, "fides-ui", "fides-pc", external=True, env=test_env_vars
     )
+    session.log("Logging in...")
+    session.run(*LOGIN, external=True)
```

Comment:
System endpoints aren't authenticated in dev mode, you need to be in prod mode.  If you're in prod mode, we need to login when running nox -s fides_env(dev)